go.mod

```diff
@@ -100,10 +103,10 @@ require (
 	golang.org/x/mod v0.21.0
 	golang.org/x/net v0.33.0
 	golang.org/x/sync v0.10.0
-	golang.org/x/sys v0.28.0
+	golang.org/x/sys v0.29.0
 	golang.org/x/time v0.6.0
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20241021214115-324edc3d5d38
-	google.golang.org/grpc v1.68.1
+	google.golang.org/grpc v1.69.4
```

Comment:
Looks like stargz has dependabot  enabled, so aggressively updating to "latest"; https://github.com/containerd/stargz-snapshotter/commit/0e94caea39fdc6a81dd9c39c971743d1ca1d61a3